Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: git://github.com/huboard/warden-github.git
-  revision: 459903681fa00820065348c27639b9f27db9fa55
+  revision: 3c3cf6943b01ff12cfe22a1ad1202ddcc4721e66
```

Comment:
Oh interesting, just for my own learning here, does that mean when looking for the warden gem, it grabs that remote and that sha on every (gem) install ?